### PR TITLE
Add [AUR] Popularity Badge

### LINKED ISSUE #10289: AUR Popularity
### 📋 Description

AUR
Popularity | 7.99

### 🔗 Data

IDK, however it is that the existing AUR badges work.

### 🎤 Motivation

Simple-icons assessing popularity rule 4.
https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#assessing-popularity

This badge would streamline pull requests for the simple icons project.

### PR DESCRIPTION
**Pull request by matinlotfali**
This closes #10289

### CODE REVIEW

File: services/aur/aur.service.js [hunk 1]
```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
...
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
+      color: floorCountColor(popularity, 0.5, 2.5, 5),
+    }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    return this.constructor.render({ votes: json.results[0].Popularity })
```
Comment (chris48s) [inline services/aur/aur.service.js:148]:
```suggestion
    return this.constructor.render({ popularity: json.results[0].Popularity })
```

`render()` does not take a parameter called `votes`
That is why every badge says `popularity | undefined`

File: services/aur/aur.service.js [hunk 2]
```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
...
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'popularity' }
+
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
+      color: floorCountColor(popularity, 0.5, 2.5, 5),
```
Comment (chris48s) [inline services/aur/aur.service.js:142]: Do these threshold numbers come from anywhere in particular, or are they just based on vibes?

File: services/aur/aur.service.js [hunk 3]
```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
...
+        parameters: pathParams({ name: 'packageName', example: 'dropbox' }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'popularity' }
+
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
```
Comment (chris48s) [inline services/aur/aur.service.js:141]: `metric()` is really intended for formatting numbers that can get quite large - it turns thousands into k, millions into M, etc.
Currently the most popular package on AUR has a rating of 38.23. I don't think this really makes sense as a formatter. I think I'd probably just display it as it stands (having been though your `.precision(2)` schema.
Lets also adjust the test assertion (`isMetric`) to match

File: services/aur/aur.service.js (hunk 1 above)
Comment (matinlotfali) [reply services/aur/aur.service.js:148]: Right! I initially copied from the `votes` and changed it all to `popularity`. I missed this one.

File: services/aur/aur.service.js (hunk 2 above)
Comment (matinlotfali) [reply services/aur/aur.service.js:142]: They are based on my own vibes.

**Comment by matinlotfali** [comment]
@chris48s can review again

File: services/aur/aur.service.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ const aurSchema = Joi.object({
       Joi.object({
         License: Joi.array().items(Joi.string().required()).allow(null),
         NumVotes: nonNegativeInteger,
+        Popularity: Joi.number().precision(2).min(0),
```
Comment (chris48s) [inline services/aur/aur.service.js:23]:
```suggestion
        Popularity: Joi.number().precision(2).min(0).required(),
```

File: services/aur/aur.tester.js [hunk 5]
```diff
@@ -35,6 +36,19 @@ t.create('votes (not found)')
   .get('/votes/not-a-package.json')
   .expectBadge({ label: 'votes', message: 'package not found' })
 
+// popularity tests
+
+t.create('popularity (valid)')
+  .get('/popularity/google-chrome.json')
+  .expectBadge({
+    label: 'popularity',
+    message: Joi.number().precision(2),
```
Comment (chris48s) [inline services/aur/aur.tester.js:45]:
```suggestion
    message: Joi.number().precision(2).required(),
```

File: services/aur/aur.service.js (hunk 2 above)
Comment (chris48s) [reply services/aur/aur.service.js:142]: Having done a bit of reading on how this is calculated/used on the AUR wiki, I don't think I have a better suggestion for a criteria

**Review by chris48s — APPROVED** [review]

**Pull request merged**